### ISSUE TITLE
Unable to recognize tab page

### ISSUE THREAD

**Issue by nbgibson** [ytdl-org/youtube-dl#27063]
## Checklist

- [X ] I'm reporting a broken site support issue
- [X ] I've verified that I'm running youtube-dl version **2020.11.18**
- [X ] I've checked that all provided URLs are alive and playable in a browser
- [X ] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [X ] I've searched the bugtracker for similar bug reports including closed ones
- [X ] I've read bugs section in FAQ

## Verbose log

```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--verbose', '--yes-playlist', '--sub-format', 'srt', '--sub-lang', 'en', '--cookies', '/home/nate/bin/cookies.txt', '-o', '/media/biggun/Plex/Youtube/%(uploader)s [%(channel_id)s]/%(playlist_index)s - %(title)s [%(id)s].%(ext)s', '--download-archive', '/home/nate/bin/watchlaterarchive.txt', 'https://www.youtube.com/playlist?list=WL']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, pref UTF-8
[debug] youtube-dl version 2020.11.18
[debug] Python version 3.8.5 (CPython) - Linux-5.4.0-54-generic-x86_64-with-glibc2.29
[debug] exe versions: ffmpeg 4.2.4, ffprobe 4.2.4
[debug] Proxy map: {}
[youtube:tab] WL: Downloading webpage
ERROR: Unable to recognize tab page; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 797, in extract_info
    ie_result = ie.extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 532, in extract
    ie_result = self._real_extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 2951, in _real_extract
    raise ExtractorError('Unable to recognize tab page')
youtube_dl.utils.ExtractorError: Unable to recognize tab page; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

## Description

Simply put ytdl appears to have some trouble handling the Watch Later tab/page. This has been the case for a few days now and may be related to recent changes from YT on how their pages are handled. Note that this was working after the recent "0 videos in playlist bug" and the associated fix.

**Comment by dstftw** [comment]
Outdated cookies.

**Comment by nbgibson** [comment]
Confirmed, thank you.
